yal/scripts/update_rapidpro_contacts_persona_emoji.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+import json
+
+import emoji
+import requests
+
+from yal.utils import extract_first_emoji
+
+
+def get_arguments() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        description="Pretends to be a webhook from whatsapp for a template message "
+        "button click with payload"
+    )
+    parser.add_argument("url", help="Base URL for the transport")
+    parser.add_argument("api_token", help="Rapidpro API token")
+
+    return parser.parse_args()
+
+
+def update_contacts(arguments: argparse.Namespace):
+    rapidpro_base_api_url = arguments.url
+    api_token = arguments.api_token
+
+    response = requests.get(
+        f"{rapidpro_base_api_url}/api/v2/contacts.json",
+        headers={"Authorization": f"Token {api_token}"},
+    )
+    response = response.json()
+
+    for contact in response["results"]:
+        if (
+            not emoji.is_emoji(contact["fields"]["persona_emoji"])
+            and contact["fields"]["persona_emoji"] != "Skip"
+            and contact["fields"]["persona_emoji"] is not None
+        ):
+            new_persona_emoji = extract_first_emoji(contact["fields"]["persona_emoji"])
```

Comment:
We should probably check if `persona_emoji == new_persona_emoji`, and if so, we can skip the request to update the contact, to avoid updating the field unnecessarily (especially since we think this applies to a small amount of contacts)

Reply:
Ah right, yeah I can't think of any values where it would make it into the if and not be changed